core/src/Kokkos_Crs.hpp

```diff
@@ -97,34 +97,69 @@ class Crs {
   typedef View<size_type* , array_layout, device_type> row_map_type;
   typedef View<DataType*  , array_layout, device_type> entries_type;
 
-  row_map_type row_map;
-  entries_type entries;
+  row_map_type row_map {};
+  entries_type entries {};
```

Comment:
Why the curlies?

Reply:
I think what Christian is saying here is that you really don't need the curlies here :-)  (Curly-brace initialization doesn't necessarily do what you think it does, so don't use it if you don't really want it.)